content/docs/02-getting-started/02-nextjs-app-router.mdx

```diff
@@ -79,6 +79,11 @@ OPENAI_API_KEY=xxxxxxxxx
 
 Replace `xxxxxxxxx` with your actual OpenAI API key.
 
+<Note className="mb-4">
+  {' '}
```

Comment:
Probably a dumb question but why is this extra space needed?

Reply:
i've been following a pre-existing pattern for usage with this component, cc'ing @jeremyphilemon for more context!

Reply:
can you check if it makes any difference in rendering? if not, i'd be in favor of removing it

Reply:
yes, can do!